engine/src/collisions/interface/collision_layers.cpp

```diff
@@ -0,0 +1,31 @@
+#include <cubos/core/ecs/reflection.hpp>
+#include <cubos/core/reflection/external/primitives.hpp>
+
+#include <cubos/engine/collisions/collision_layers.hpp>
+
+CUBOS_REFLECT_IMPL(cubos::engine::CollisionLayers)
+{
+    return core::ecs::TypeBuilder<CollisionLayers>("cubos::engine::CollisionLayers")
+        .withField("value", &CollisionLayers::value)
+        .build();
+}
+
+void cubos::engine::CollisionLayers::setLayerValue(unsigned int layerNumber, bool newValue)
+{
+    if (layerNumber > 31)
+    {
+        return;
```

Comment:
I would either add an assert here or log an error. This kind of thing shouldn't fail silently, I think